apps/jobs/views.py

```diff
@@ -485,7 +489,7 @@ def leaderboard(request, challenge_phase_split_id):
 
     sorted_leaderboard_data = sorted(
         leaderboard_data,
-        key=lambda k: float(k["filtering_score"]),
+        key=lambda k: (float(k["filtering_score"]), float(-k["filtering_error"])),
```

Comment:
What does this line do?

Reply:
This will sort leaderboard_data according to `filtering_score` as well as `filtering_error`. If `filtering_score` is same, `filtering_error` will be used to break ties, in increasing order. 
Please see `total` in screenshot below, where the `total` is same for both but tie is resolved using error.
![Screenshot from 2019-04-17 03-57-57](https://user-images.githubusercontent.com/3920286/56545124-2f9e9d00-6594-11e9-9200-c1725c0cb60b.png).
If the leaderboard has no error bar, `filtering_error` is default to `0`, to avoid `None Type` error.

Reply:
Cool, thanks!